.azure/scale.csv

```diff
@@ -0,0 +1,7 @@
+stage,siteId,location,domainFqdn
```

Comment:
## Arc IaC Automation generated
Next steps: 
  - [ ] Fill the csv file and commit, the struct of the csv file can be found on [here](https://github.com/Azure/Edge-infrastructure-quickstart-template/blob/main/doc/Scale-with-automation.md#methodology)
  - [ ] Wait for the scale workflow run, and then resolve this comment and merge the pull request

 If you have any problem, please file an [issue](https://github.com/Azure/Edge-infrastructure-quickstart-template/issues/new/choose) or contact arcIaCSupport@microsoft.com